assets/js/googlesitekit/datastore/user/user-input-settings.js

```diff
@@ -248,7 +246,7 @@ const baseResolvers = {
 			yield fetchGetUserInputSettingsStore.actions.fetchGetUserInputSettings();
 		}
 
-		if ( select( CORE_USER ).getUserInputState() !== 'completed' ) {
+		if ( ! select( CORE_USER ).isUserInputCompleted() ) {
 			yield baseActions.setUserInputSettingsFromCache();
```

Comment:
@kuasha420 this strikes me as something that we should be removing as user input settings come from the local site now. Is this removed in another issue? If not, let's open one for addressing this.

Reply:
Hi, good catch, we had #5899 as a catch-all issue to clean things up but due to how everything panned out, that one got closed. I'll create a new issue to handle this. :+1: